build/tests/test_cli.py

```diff
@@ -18,6 +18,9 @@ def test_models(model_constructor, tmp_path):
     assert isinstance(model, ribasim.Model)
     model.write(tmp_path / "ribasim.toml")
 
+    if model.experimental.allocation:
```

Comment:
Ideally we mark these tests with `xfail`: https://docs.pytest.org/en/stable/how-to/skipping.html#skip-xfail-with-parametrize